src/attachment/index.ts

```diff
@@ -189,6 +201,38 @@ let exportAttachment;
         </li>
       `;
     }
+    async updated(_changedProperties: PropertyValues) {
+      await this.updateComplete;
+      this._updateFileNameMaxWidth();
+    }
+
+    private _updateFileNameMaxWidth() {
```

Comment:
You can change the CSS file to adjust the width of root.
And only adjust the width of file name in this function.